docs/Shell-Completion.md

```diff
@@ -24,6 +24,22 @@ fi
 ```
 
 Should you later install the `bash-completion` formula, this will automatically use its initialization script to read the completions files.
+When installed, the `bash-completion` formula also runs `${HOMEBREW_PREFIX}/etc/profile.d/bash_completion.sh` and all files in the `bash_completion.d` directory. This is done by adding a line to your `.bash_profile` -
+
+```
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
+```
+which is mentioned in the Caveats section upon the installation of `bash-completion`
+
+As both Homebrew's completion code given above and the Caveats line do the same thing, it is recommended to either not add the Caveats line or to comment the line out because Homebrew's completion code works even without installing the `bash-completion` formula.
+
+---
+**Note**
+
+If you are using a version of `bash` newer than version 4.1 (Like Homebrew's `bash`), it is recommended to use the `bash-completion@2` formula instead as it is newer and has better performance. You can check the version of `bash` you have by running `bash --version`. MacOS ships with version 3.2.
```

Comment:
"Like" and "MacOS" don't need to be capitalized. Also, use active voice to keep instructions clear and concise.

Reply:
Done! Are you sure we need to un-capitalize "MacOS"? It is the start of a sentence, right?

Reply:
`macOS` is a proper noun so shouldn't be capitalised.